### DISCUSSION TITLE
Upload data error - HNN gui

### DISCUSSION THREAD

**Discussion by sarapiers** [jonescompneurolab/hnn-core#665] [Q&A] [answered]
I am new to this, so I hope not to waste too much of your time, and I am trying to use the hnn-gui to plot my own source localized data but I am encountering the following issues:

1) When trying to upload my data it gives me an error on window lenght (even if I changed the tstop and dt according to my data):
"ValueError: Window length too long: 15000.0 samples; data length is 2101 samples."

When creating the .txt file I followed the instructions on https://jonescompneurolab.github.io/hnn
-tutorials/starting_with_data/getting_started and it is not specified if I have to make sure to have a certain number of data to plot, and do I also have to upload external drives at the same time?

2) I can't delete the uploaded data, if I upload new ones it appears like I have 2 uploaded file at the same time (but still since there is an error, I think hnn is not reading any of them and just running the defoult simulation).

3) I can't exit hnn from anaconda, neither deactivate the environment, if I want to exit hnn then I have to shut off anaconda and turn it on again.

Thank you for your help,
Sara

**Comment by rythorpe** [comment]
The window length error makes me suspect it's trying to convolve your experimental current dipole data with a smoothing window or wavelet function. Any thoughts here @chenghuzi?

**Comment by jasmainak** [comment, marked as answer]
As discussed on email, the time column should be in milliseconds rather than seconds.
